go/webpack/cbor_format_test.go

```diff
@@ -31,10 +31,14 @@ func TestWriteCBOR(t *testing.T) {
 	pack := Package{
 		parts: []*PackPart{
 			&PackPart{
-				url:            staticUrl("https://example.com/index.html?query"),
-				requestHeaders: HTTPHeaders{},
-				status:         200,
+				requestHeaders: HTTPHeaders{
+					httpHeader(":method", "GET"),
```

Comment:
I'm not sure we need `:method`: it's here because it's in HTTP2, but I don't know of any methods besides GET that we'd want to support. I think this is something the folks at the IETF might be able to give us useful feedback about. I've filed #70 to track this.